libs/application/templates/parental-leave/src/lib/ParentalLeaveTemplate.ts

```diff
@@ -561,6 +571,7 @@ const ParentalLeaveTemplate: ApplicationTemplate<
         entry: 'assignToVMST',
         meta: {
           name: States.APPROVED,
+          status: 'approved',
```

Comment:
Status is still 'inprogress' here. Even VMST has approved application. Applicant could still add information/make changes and send it in again